components/client/typescript/src/schemas/bitcoin/payload.ts

```diff
@@ -33,7 +33,27 @@ export const BitcoinInscriptionTransferredSchema = Type.Object({
 });
 export type BitcoinInscriptionTransferred = Static<typeof BitcoinInscriptionTransferredSchema>;
 
+export const BitcoinCursedInscriptionRevealedSchema = Type.Object({
+  content_bytes: Type.String(),
+  content_type: Type.String(),
+  content_length: Type.Integer(),
+  inscription_number: Type.Integer(),
+  inscription_fee: Type.Integer(),
+  inscription_id: Type.String(),
+  inscription_output_value: Type.Integer(),
+  inscriber_address: Type.String(),
```

Comment:
Should we update this to accept optionals?

Reply:
Done!